public/docs/ts/latest/quickstart.jade

```diff
@@ -10,479 +10,471 @@ include ../../../_includes/_util-fns
+  giving the matter some thought.
+
+  If we set the `noImplicitAny` flag to `true`, we may get implicit index errors as well.
+  If we feel these are more annoying than helpful,
+  we can suppress them with the following additional flag.
+  ```
+  "suppressImplicitAnyIndexErrors":true
+  ```
 
 .l-main-section
-  :marked
-    <a id="es6support"></a>
-    ### Appendix: Browser ES6 support
-    Angular 2 requires ES6 support, such as can be found in most modern
-    browsers. For older browsers (including IE 11) you can use a shim to get
-    the needed functionality.
-
-    After creating `package.json` (halfway through the quickguide), run this
-    command to add a shim to the project:
-
-  code-example(language="sh" format=".").
-    npm install es6-shim --save
-
-  :marked
-    Now you can load the shim in your `index.html` before the other scripts:
-
-  code-example(language="html" format=".").
-    &lt;script src=&quot;../node_modules/es6-shim/es6-shim.js&quot;&gt;&lt;/script&gt;
+:marked
+  <a id="systemjs"></a>
+  ### Appendix: SystemJS Configuration
+  
+  The QuickStart uses [SystemJS](https://github.com/systemjs/systemjs) to load application
+  and library modules. 
+  There are alternatives that work just fine including the well-regarded [webpack](https://webpack.github.io/).
+  SystemJS happens to be a good choice but we want to be clear that it was a choice and not a preference.
+  
+  All module loaders require configuration and all loader configuration 
+  becomes complicated rather quickly as soon as the file structure diversifies and
+  we start thinking about building for production and performance.
+  
+  We suggest becoming well-versed in the loader of your choice.
+  
+  With those caustions in mind, what are we doing here?
++makeExample('quickstart/ts/index.html', 'systemjs', 'index.html (System configuration')(format=".")
+:marked
+  The `packages:` line tells SystemJS what to do when it sees a request for a 
+  module from the `app/` folder. 
+  
+  Our QuickStart makes such requests when one of its
+  application TypeScript files has an import statement like this:
++makeExample('quickstart/ts/app/boot.ts', 'app-component', 'boot.ts (excerpt)')(format=".")
+:marked
+  Notice that the filename has not extension. 
+  The `packages:` line tells SystemJS to default the extension to 'js', a JavaScript file.
+  
+  That makes sense when you know, as we do, that we transpile TypeScript to JavaScript
+  *before* running the application.
+  
+  The `System.Import` line tells SystemJS to import the `boot` file 
+  (`boot.js` ... after transpiling `boot.ts`, remember?).
+  
+  That's where we *bootstrap* the application. All other modules are loaded upon request
+  either by an import statement or by Angular itself.
 
+.l-main-section
+:marked
+  <a id="es6support"></a>
+  ### Appendix: Browser ES6 support
+  Angular 2 requires ES6 support, such as can be found in most modern
```

Comment:
this is sort of ambiguous and broad (and scary) - what about something like "angular2 uses some ES6 language features..."